README.md

```diff
@@ -27,4 +27,5 @@ require'telescope'.extensions.dap.commands{}
 require'telescope'.extensions.dap.configurations{}
 require'telescope'.extensions.dap.list_breakpoints{}
 require'telescope'.extensions.dap.variables{}
+require'telescope'.extensions.dap.frames{}
```

Comment:
We can also add `Telescope dap frames` etc to readme because thats a thing now and i forgot to update it here :)

Reply:
I don't get that one. I just documented the  `frames` function in the same way the other functions were listed

Reply:
Since i wrote this extension we had changes in telescope core and we are now picking up extension with the vim `:Telescope` command. What you did here was good but i was wondering if we want to add the viml side as well since its possible now. So
```viml
:Telescope dap commands
:Telescope dap configurations
...
```

Reply:
One could add autocomplete for that commands...

Reply:
We have but its kinda buggy :laughing: And i think you need to have `require('telescope').load_extension('dap')` set.

Just needs a little bit more work